travis-ci-inspired-dark-extended.css

```diff
@@ -1,4 +1,4 @@
-@-moz-document domain("travis-ci.org") {
+@-moz-document domain("travis-ci.org"), domain("travis-ci.com") {
```

Comment:
I'm not totally sure what the distinction is between .com and .org, but I'm guessing the former is paid and the latter is not, or something along those lines. I was seeing issues because I use travis-ci.com at work and many of these rules were not getting applied. A cursory exploration of the two domains open side-by-side doesn't bring up any glaring differences with these changes, and I would be very surprised if they were using these classes differently in such a way as to break theming like this.

In any case: it's not directly related to the class rename, but it is still an issue because some themes were missing on the .com.

I can get some side-by-side screenshots if that would help, but IIRC it's pretty obvious if you open the page that something is missing.

Reply:
HI.

Yes .com is paid and .org is not.

I will need to have a quick test at some stage to ensure no other areas are being affected before merging, I would be curious if instead of renaming them we would instead remove this lot, since these classes should be automatically loaded from default stylesheet, probably need some browser cache clearing if so.

I will try and see if I can get a few minutes in the next few days to that effect. so stay tuned and thank you in advance for your patience with any delays getting back to you.

:)

Reply:
> I would be curious if instead of renaming them we would instead remove this lot, since these classes should be automatically loaded from default stylesheet

Not totally sure what you mean here: it seems they're necessary because otherwise you get the blinding white! But I'll wait for your further response.

Also, if it's easier, I can split this PR into two: one for the rename and one for the .com addition.

Reply:
Hi there. Ive had a short opportunity to test this minimally, it seems quite a lot more changes happened and there are a great deal more areas that broke aside from these not just within .com but also .org. for instance https://travis-ci.org/Atom

>Also, if it's easier, I can split this PR into two: one for the rename and one for the .com addition.

Yes please, it would make it slightly easier in case something else breaks.

>  Not totally sure what you mean here: it seems they're necessary because otherwise you get the blinding white! But I'll wait for your further response.

Sorry for any confusion, when I originally did this, a great deal of selectors were not needed as I copied them from default stylesheets, and I suspect despite some cleanups, more will be needed in due time. No worries though.